source/documentation/administration_guide/chap-Renewing_certificates.adoc

```diff
@@ -7,7 +7,7 @@
 
 [WARNING]
 ====
-Do _NOT_ let certificates expire. If they expire, the environment becomes non-responsive and recovery is an error prone and time consuming process. If they do expire, you can manually renew them by following the instructions in link:https://access.redhat.com/solutions/3532921[How to manually renew RHV host SSL certificate if expired?]. Once you have completed the steps to manually renew the certificates, follow the steps in this procedure to properly enroll {virt-product-shortname} host certificates.
+Do not let certificates expire. If they expire, the environment becomes non-responsive and recovery is an error-prone and time-consuming process. If they do expire, you can manually renew them by following the instructions in link:https://access.redhat.com/solutions/3532921[How to manually renew RHV host SSL certificate if expired?]. Once you have completed the steps to manually renew the certificates, follow the steps in this procedure to properly enroll {virt-product-shortname} host certificates.
 ====
 
 .Procedure
```

Comment:
I think step 2 in this procedure looks weird. It's not really a step. It's what happens automatically when you put the host into maintenance mode.

@sandrobonazzola Can you confirm?

Reply:
well, yes. but they can also be set as pinned. It's worth elaborating a bit more in this step